WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewController.swift

```diff
@@ -1,9 +1,91 @@
 import UIKit
+import Combine
+
+final class CardReaderSettingsViewController: UIViewController {
+    private var subscriptions = Set<AnyCancellable>()
+    private var viewmodel = CardReaderSettingsViewModel()
+    private var subView: UIView?
+    private var alert: UIAlertController?
```

Comment:
I am not sure it is necessary to retain the alert view controller, specially if it is a FancyAlertViewController.

Reply:
I am retaining it so, if/when the VM says it can go (e.g. searching finds a reader), the VC can dismiss it